Proposals/xgov-26.md

```diff
@@ -0,0 +1,211 @@
+---
+title: ASA Stats Offer - Algorand blockchain ASA/NFT creators' channel
+author: Ivica Paleka (@ipaleka), Marcin Zawiejski (@dragmz), Eduard Ravnic (@kerrilija)
+company_name: ASA Stats
+category: Tools
+focus_area: User Onboarding
+open_source: Yes
+amount_requested: 250000
+status: Draft
```

Comment:
Remember to change this to "Final" if you want your proposal to go through the next voting session.